force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -564,6 +564,21 @@ export default class rd2EntryForm extends LightningElement {
             payload: event.target.value,
         });
     }
+    
+    experienceAmountValidation(event) {
+
+        if(this.isExperienceSite) {
+            let newValue = event.target.value;
+            if(/^[0-9]+$/.test(String.fromCharCode(event.keyCode))) {
+                newValue += String.fromCharCode(event.keyCode);
+            }
+            if(newValue < 1 || newValue > 99999999) {
```

Comment:
Is `99999999` an arbitrary numeric value to check for, or is there some defined upper limit for a gift?

Reply:
@force2b based on [this documentation](https://salesforce.quip.com/kRYaAjEE8aHr#temp:C:PQJ10eb9e91d1fe4222ab6c78de9)

Reply:
Thanks.